lib/cloudware/config.rb

```diff
@@ -28,32 +30,60 @@ class Config
     attr_accessor :aws_access_key_id, :aws_secret_access_key
     attr_accessor :providers
 
-    def initialize(cfg_file)
-      config = YAML.load_file(cfg_file) || raise("Couldn't load config file #{cfg_file}")
+    def initialize
+      config = YAML.load_file(config_path) || raise("Couldn't load config file #{cfg_file}")
```

Comment:
Doesn't look like `#{cfg_file}` is actually set, should this be `#{config_path}`?

Reply:
There is a private `config` method at the end of the file

Reply:
I'm aware that that's what's used to load the file path. What I'm addressing here is the `#{cfg_file}` at the end of `raise("Couldn't load config file #{cfg_file}")`, I don't believe this variable is not set anywhere unless I'm mistaken?

Reply:
Ohh I see. I have fixed the issue on my current working branch. For the time being leave it, its a poor check anyway. It will only ever raise if the file is present but completely empty. If the file is missing the `YAML` will error. If the file contains any content (except `false`) it will pass.